php/libraries/Utility.class.inc

```diff
@@ -539,7 +539,7 @@ class Utility
      */
     static function toArray($var)
     {
-        if (!isset($var[0])) {
+        if (!array_key_exists(0, $var)) {
```

Comment:
Never mind you've already done it. Github needs to make improvements on this review/commenting thing as it shows the code for previous commit. It would be nice if the diff was of the commit that the comment was submitted on and the latest version of the pull request